diff --git a/scripts/trackbar.pl b/scripts/trackbar.pl
index 7ac9965..86c690b 100644
--- a/scripts/trackbar.pl
+++ b/scripts/trackbar.pl
@@ -99,7 +99,7 @@ use strict;
 use warnings;
 use vars qw($VERSION %IRSSI);
 
-$VERSION = "2.5"; # 56e983314dc1b85
+$VERSION = "2.6"; # 1467eeaa92d28bf
 
 %IRSSI = (
     authors     => "Peter 'kinlo' Leurs, Uwe Dudenhoeffer, " .
@@ -331,6 +331,7 @@ sub sig_window_changed {
     return if delete $keep_trackbar{ $oldwindow->{_irssi} };
     trackbar_update_seen($oldwindow);
     return if $config{require_seen} && $unseen_trackbar{ $oldwindow->{_irssi } };
+    return if $config{all_manual};
     update_one_trackbar($oldwindow, undef, 0);
 }
 
@@ -515,6 +516,7 @@ sub update_config {
     $config{style} = Irssi::settings_get_str('trackbar_style');
     $config{string} = Irssi::settings_get_str('trackbar_string');
     $config{require_seen} = Irssi::settings_get_bool('trackbar_require_seen');
+    $config{all_manual} = Irssi::settings_get_bool('trackbar_all_manual');
     $config{ignore_windows} = [ split /[,\s]+/, Irssi::settings_get_str('trackbar_ignore_windows') ];
     $config{use_status_window} = Irssi::settings_get_bool('trackbar_use_status_window');
     $config{print_timestamp} = Irssi::settings_get_bool('trackbar_print_timestamp');
@@ -545,6 +547,7 @@ Irssi::settings_add_str('trackbar', 'trackbar_ignore_windows', '');
 Irssi::settings_add_bool('trackbar', 'trackbar_use_status_window', 1);
 Irssi::settings_add_bool('trackbar', 'trackbar_print_timestamp', 0);
 Irssi::settings_add_bool('trackbar', 'trackbar_require_seen', 0);
+Irssi::settings_add_bool('trackbar', 'trackbar_all_manual', 0);
 
 update_config();
